### PR TITLE
Pull in latest changes from Tractus-X

### PR DESCRIPTION
**Pull request by ds-jkreutzfeld**
Our main branch will be updated with the latest changes from Tractus-X.
Review the changes to identify any potential problems (API changes, etc).

After the merge, we will try to update dev/ as well.

### CODE REVIEW

File: frontend/src/app/modules/page/alerts/presentation/alerts.component.html [hunk 1]
```diff
@@ -28,9 +28,11 @@
   [receivedNotifications$]="alertsReceived$"
   [translationContext]="TranslationContext.COMMONALERT"
   [receivedOptionalColumns]="['severity', 'createdBy']"
-  [receivedSortableColumns]="{status: true, createdDate: true, severity: true, createdBy: true}"
+  [receivedMultiSortList]="alertReceivedSortList"
+  [receivedSortableColumns]="{description: true, status: true, createdDate: true, severity: true, createdBy: true}"
   [queuedAndRequestedOptionalColumns]="['severity', 'sendTo']"
-  [queuedAndRequestedSortableColumns]="{status: true, createdDate: true, severity: true, sendTo: true}"
+  [queuedAndRequestedSortableColumns]="{description: true, status: true, createdDate: true, severity: true, sendTo: true}"
```
Comment (viannbreslerAccso) [inline frontend/src/app/modules/page/alerts/presentation/alerts.component.html:34]: There will be some conflicts here, it seems they have changed the way that columns are displayed. Shouldn't be too complex to resolve

File: frontend/src/app/modules/page/investigations/presentation/investigations.component.html [hunk 2]
```diff
@@ -30,9 +30,11 @@
   [receivedNotifications$]="investigationsReceived$"
   [translationContext]="TranslationContext.COMMONINVESTIGATION"
   [receivedOptionalColumns]="['targetDate', 'severity', 'createdBy']"
-  [receivedSortableColumns]="{status: true, createdDate: true, severity: true, createdBy: true}"
+  [receivedSortableColumns]="{description: true, status: true, createdDate: true, severity: true, createdBy: true, targetDate: true}"
   [queuedAndRequestedOptionalColumns]="['targetDate', 'severity', 'sendTo']"
-  [queuedAndRequestedSortableColumns]="{status: true, createdDate: true, severity: true, sendTo: true}"
+  [queuedAndRequestedSortableColumns]="{description: true, status: true, createdDate: true, severity: true, sendTo: true, targetDate: true}"
```
Comment (viannbreslerAccso) [inline frontend/src/app/modules/page/investigations/presentation/investigations.component.html:35]: Same type of change regarding the way table columns are displayed

**Comment by viannbreslerAccso** [comment]
There will be a few conflicts on the FE, but these won't be too difficult to resolve. Luckily most of the changes are in files we haven't touched yet

**Review by viannbreslerAccso — APPROVED** [review]

**Review by hgo-ds — APPROVED** [review]

**Comment by hgo-ds** [comment]
Followup to merge into dev-branch, see PR #16

**Comment by hgo-ds** [comment]
Followup merge from upstream into main: PR #140

**Pull request merged**